docs/core/compatibility/aspnet-core/5.0/authentication-aad-packages-obsolete.md

```diff
@@ -1,36 +1,39 @@
-### Authentication: AzureAD.UI and AzureADB2C.UI APIs and packages marked obsolete
+---
+title: "Breaking change: Authentication: AzureAD.UI and AzureADB2C.UI APIs and packages marked obsolete"
+description: "Learn about the breaking change in ASP.NET Core 5.0 titled Authentication: AzureAD.UI and AzureADB2C.UI APIs and packages marked obsolete"
+author: scottaddie
+ms.author: scaddie
+ms.date: 10/01/2020
+---
+# Authentication: AzureAD.UI and AzureADB2C.UI APIs and packages marked obsolete
 
 In ASP.NET Core 2.1, integration with Azure Active Directory (Azure AD) and Azure Active Directory B2C (Azure AD B2C) authentication is provided by the [Microsoft.AspNetCore.Authentication.AzureAD.UI](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.AzureAD.UI) and [Microsoft.AspNetCore.Authentication.AzureADB2C.UI](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.AzureADB2C.UI) packages. The functionality provided by these packages is based on the Azure AD v1.0 endpoint.
 
 In ASP.NET Core 5.0 and later, integration with Azure AD and Azure AD B2C authentication is provided by the [Microsoft.Identity.Web](https://www.nuget.org/packages/Microsoft.Identity.Web) package. This package is based on the Microsoft Identity Platform, which is formerly known as the Azure AD v2.0 endpoint. Consequently, the old APIs in the `Microsoft.AspNetCore.Authentication.AzureAD.UI` and `Microsoft.AspNetCore.Authentication.AzureADB2C.UI` packages were deprecated.
 
 For discussion, see GitHub issue [dotnet/aspnetcore#25807](https://github.com/dotnet/aspnetcore/issues/25807).
 
-#### Version introduced
+## Version introduced
 
 5.0 Preview 8
 
-#### Old behavior
+## Old behavior
 
 The APIs weren't marked as obsolete.
 
-#### New behavior
+## New behavior
 
 The APIs are marked as obsolete.
 
-#### Reason for change
+## Reason for change
```

Comment:
I already pointed to that in the previous PR, but just as a reminder, this breaks the apiport tool.

That will need to be fixed before merging `release-breaking-changes` into master.

Reply:
@Youssef1313 Thanks, I thought you were just saying that "Category" and "Affected APIs" were required to be H3s, but now I see it's all the headings. But in that case, I don't see how the tool is working even with the existing format where we use H4 headings for these headings. The code appears to be expecting H3s.

```csharp
if (currentLine.StartsWith("###", StringComparison.Ordinal))
{
    switch (currentLine.Substring("###".Length).Trim().ToLowerInvariant())
    {
        case "scope":
            state = ParseState.Scope;
            break;
        case "version introduced":
        case "version broken":
            state = ParseState.VersionBroken;
            break;
        ...
```

@Lxiamail Does the ApiPort tool even work with the existing format? Who should I talk to about updating the tool to work with this new format where each breaking change is in a separate file, and the headings to parse are H2s?

Reply:
Yeah I didn't notice it was expecting H3 but the actual was H4. So it looks like it's already broken for a while? not sure.